Modules/_elementtree.c

```diff
@@ -2575,14 +2575,24 @@ xmlparser(PyObject* self_, PyObject* args, PyObject* kw)
         return NULL;
     }
 
+    ALLOC(sizeof(XMLParserObject), "create expatparser");
+
+    /* Init to NULL to keep the error handling below manageable. */
+    self->target =
```

Comment:
This is not needed. `PyObject_New()` clears the memory.

Reply:
I thought so, too, until the test started crashing due to uninitalised struct fields.

Reply:
Oh, seems I remember this wrong.